metadata/src/test/java/org/apache/kafka/image/loader/MetadataLoaderTest.java

```diff
@@ -597,7 +597,7 @@ private void loadTestSnapshot2(
                 new MetadataProvenance(offset, 100, 4000, true), asList(
                         singletonList(new ApiMessageAndVersion(new FeatureLevelRecord().
                                 setName(MetadataVersion.FEATURE_NAME).
-                                setFeatureLevel(IBP_3_3_IV2.featureLevel()), (short) 0)),
+                                setFeatureLevel(IBP_3_4_IV0.featureLevel()), (short) 0)),
```

Comment:
Since this can be any MV larger than MetadataVersion.MINIMUM_VERSION, could we just use MetadataVersion.latestProduction?